SR2020/2020-07-05-virtual-competition-timings.md

```diff
@@ -0,0 +1,20 @@
+---
+to: sr2020-teams
+subject: SR2020 Virtual Competition Schedule
+---
+
+Hi all
+
+We're pleased to announce the structure of the SR2020 virtual competition. We will be running 5 sessions over 3 weekends.
+
+You will have until 10am on each day to make your changes before your team's code is frozen. Then **at 3pm we will be live-streaming matches** back-to-back on YouTube.
```

Comment:
I think we should document here *how* submission will work. Namely, saying we'll take a project on the IDE named `virtual-competition`, and download it at 10am.

Reply:
There's a couple of considerations around this which we'd not previously discussed that lead to us being less confident about an IDE solution. I think it could still be a valid fallback, but IMO leaving this vague for now gets us more time to find an alternative (I'm hoping to get my submitter thing working this weekend).

I'll put the discussion details on https://github.com/srobo/tasks/issues/593.

Reply:
I think it's worth mentioning that the plan is to let them know later